src/app/pools/views/PoolList/PoolList.tsx

```diff
@@ -82,40 +80,29 @@ const generateRows = (rows: ResourcePool[]) =>
 
 const Pools = (): JSX.Element => {
   useWindowTitle("Pools");
-  const dispatch = useDispatch();
 
-  const poolsLoaded = useSelector(resourcePoolSelectors.loaded);
-  const poolsLoading = useSelector(resourcePoolSelectors.loading);
-  const errors = useSelector(resourcePoolSelectors.errors);
-  const errorMessage = formatErrors(errors);
-
-  useFetchActions([resourcePoolActions.fetch]);
-
-  const resourcePools = useSelector(resourcePoolSelectors.all);
+  const listPools = useListPools();
+  const errorMessage = formatErrors(listPools.error);
+  const listPoolsResources = listPools.data?.items || [];
 
   return (
     <div aria-label={Label.Title}>
       {errorMessage ? (
         <Row>
           <Col size={12}>
-            <Notification
-              onDismiss={() => dispatch(resourcePoolActions.cleanup())}
-              severity="negative"
-            >
-              {errorMessage}
-            </Notification>
+            <Notification severity="negative">{errorMessage}</Notification>
           </Col>
         </Row>
       ) : null}
       <Row>
         <Col size={12}>
           <div>
-            {poolsLoading && (
+            {listPools.isLoading && (
```

Comment:
```suggestion
            {listPools.isPending && (
```

We should use `isPending` instead of `isLoading`: https://github.com/TanStack/query/discussions/6297#discussioncomment-7467010